src/main/equationServlet.java

```diff
@@ -0,0 +1,120 @@
+package ru.itmo.SergeyPesegov.web.servlets;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.*;
+
+@WebServlet(
+        urlPatterns = {"/calc"}
+)
+public class equationServlet extends HttpServlet {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class name 'equationServlet' doesn't match '[A-Z][a-zA-Z0-9]*'](https://app.codacy.com/app/thejerome/IFMO_Servlets_20190915/pullRequest?prid=4302155)